governance/process.md

```diff
@@ -1,144 +1,209 @@
+   it is beneficial to present the work to the broader group to gather a final
+   review, solicit additional feedback, or seek recommendations. These should be
+   placed in the planned meetings section with a project member as the point of
+   contact to present.
+
+1. **Pull Requests.** Completed work should result in a Pull Request (PR). At
+   minimum, an update to one of the group documents or roadmap indicating that
+   the work was done. Typically projects will result in an artifact that will
+   contribute to the information in this repository.
 
 1. **Discuss the work at a meeting.** If an objection to a PR is made either in
-a comment of the PR or during a meeting, the person making the objection and
-the person making the proposal will be given time to present their view at the
-next meeting. If there are not objections, or if all concerns have been
-addressed, and the Pull Request has been stable for 24 hours, a Chair will add
-it to the agenda for an upcoming meeting. Ideally, members who contributed to
-the project will attend that meeting to present their work or answer questions.
-
-1. **Vote, if required.** In some cases, there's consensus to accept a
-proposal, and a vote is not required. If there’s not consensus among the group,
-a formal vote is required. A comment will be left on the proposal prompting
-members to vote and indicating the time the vote will close. Only one member
-from each company should vote. Members will vote by leaving a comment in the
-Pull Request to indicate their vote for or against. Members will have a week
-after the vote begins to leave their vote. Quorum is taken to be 2/3 the number
-of companies who have been active in the past month (via issue comment or
-meeting attendance as recorded in the public minutes).
+   a comment of the PR or during a meeting, the person making the objection and
+   the person making the proposal will be given time to present their view at
+   the next meeting. If there are not objections, or if all concerns have been
+   addressed, and the Pull Request has been stable for 24 hours, a Chair will
+   add it to the agenda for an upcoming meeting. Ideally, members who
+   contributed to the project will attend that meeting to present their work or
+   answer questions.
+
+1. **Vote, if required.** In some cases, there's consensus to accept a PR, and a
+   vote is not required. If there’s not consensus among the group, a formal vote
+   is required. A comment will be left on the PR prompting members to vote and
+   indicating the time the vote will close. Only one member from each company
+   should vote. Members will vote by leaving a comment in the Pull Request to
+   indicate their vote for or against. Members will have a week after the vote
+   begins to leave their vote. Quorum is taken to be 2/3 the number of companies
+   who have been active in the past month (via issue comment or meeting
+   attendance as recorded in the public minutes).
```

Comment:
I'm a little unsure about the voting system, I think we haven't had a situation where this is needed yet, and wondering if it would be best to hold off on this until we see the need to... I think leaving it to the decision of leadership (or a unanimous decision among leadership) would be good enough.

Reply:
If we do decide to go with voting, I think that we need to have a bit more "system" around this. Voting needs to ensure visibility of the vote happening, maybe via email list. There needs to be proof of participation provided with the vote so we don't have to go around digging up history. We need to ensure that there isn't bad faith with editing meeting notes to include past participation since there is now an incentive to do so.

Reply:
I agree with Brandon that the final decision should be left with the leadership team. We could do a majority or 2/3rd vote in the leadership team. We can encourage community members to submit "non-binding" votes to show support or lack of it.